src/Ops.Web/Areas/SiteManagement/Views/Locations/LocationDetails.cshtml

```diff
@@ -660,19 +1022,19 @@
             var addressdata = JSON.stringify(data);
 
             $.get("@Url.Action(nameof(LocationsController.GetUrlLocations))", { addressJson: addressdata }, function (response) {
-                if (response.success == true) {
-                    var locations = $.parseJSON(response.data);
-                    HandleGoogleLocations(locations);
+            if (response.success == true) {
+            var locations = $.parseJSON(response.data);
+            HandleGoogleLocations(locations);
```

Comment:
Not sure if the document wasn't formatted or, if these just need to be indented for readability.